src/main/resources/reference.conf

```diff
@@ -190,6 +190,9 @@ cassandra-journal {
   # Dispatcher for the plugin actor.
   plugin-dispatcher = "cassandra-journal.default-dispatcher"
 
+  # The query journal to use when recoverying
+  recovery-cassandra-query-journal = "cassandra-query-journal"
+
```

Comment:
ya, certainly would be better to enforce the relationship between the journal and query plugin via the configuration but assume we need to wait for 19822 to be completed first. 

In the interim I have no problem in renaming the config to query-plugin. Will do that now.